docs/modules/ROOT/pages/index.adoc

```diff
@@ -83,6 +83,8 @@ image::overview-multi-author.drawio.svg[Multiple identity assertion example,widt
 
 The following is a non-exhaustive list of potential and general use cases for the *<<_identity_assertion,identity assertion>>.* Some of these are taken from, or built upon, the use cases developed within the link:https://www.originproject.info/[Project Origin Alliance] and the link:https://contentauthenticity.org/[Content Authenticity Initiative (CAI)] frameworks. Each use case will be described using some generic personas to help make the flow clear.
 
+IMPORTANT: The identity assertion should not be construed to convey _ownership_ of a _<<C2PA asset>>._ Ownership and rights transfers are often conducted outside of the digital realm. There should be no expectation that such transactions could be accurately reflected and updated in this assertion.
```

Comment:
I wonder if it is worth adding a note here something like:

NOTE: While this assertion cannot be used to convey ownership or rights, some other assertion could convey that information, and it could be included in the set of "referenced assertions".